src/consensus/consensus.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2014 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_CONSENSUS_H
```

Comment:
This has to be ``BITCOIN_CONSENSUS_CONSENSUS_H``, because you are in the directory consensus.